frontend/src/hooks/useLanguage.js

```
@@ -601,6 +801,305 @@ const translations = {
+    sessionsToday: '今日專注次數', generatingPlan: '正在生成您的計劃...', noPendingTasks: '沒有待處理任務！',
+    addTasksFromDashboard: '從儀表板添加任務', loadingCalendar: '正在加載日曆...',
+    refresh: '刷新', tasksPending: '待處理任務', completedToday: '今日已完成',
+    recommendedTaskOrder: '推薦順序', noTasks: '暫無任務', askMeAnything: '問我任何問題...',
+    send: '發送', noLanguagesFound: '未找到語言', enterYourEmail: '您的郵箱',
+    enterPassword: '密碼', defaultDuration: '60',
+  },
+  ja: {
+    dashboard: 'ダッシュボード', tasks: 'タスク', weekly: '毎週', today: '今日',
+    emailSync: 'メール同期', addTask: '+ タスク追加', profile: 'プロフィール',
+    logout: 'ログアウト', changePassword: 'パスワード変更', welcome: 'ようこそ',
+    yourTasks: 'あなたのタスク', noTasks: 'タスクなし', addNewTask: '新しいタスク追加',
```

Biome (2.4.7)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)